lesson9/index.js

```diff
@@ -0,0 +1,46 @@
+const ul = document.getElementById('js-lists');
+
+const listData = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+function displayLoadingImage () {
+    const lodingImage = document.createElement('img');
+    lodingImage.id = 'loadingImageId';
+    lodingImage.src = './loading-circle.gif';
+    ul.appendChild(lodingImage);
+}
+
+async function loadingPromise() {
+    return new Promise(resolve => {
+        displayLoadingImage ();
+        setTimeout(() => {
+            resolve(listData);
+            const lodingImage = document.getElementById('loadingImageId');
+            lodingImage.remove();
```

Comment:
[nits]
I thought it would be easier to read if you made it a function here too.

Reply:
Thank you for your reviewing!!
I created new function `removeLoadingImage`.
I think it's easier to read!!
commit: 1884d1e